upgrade_test.py

```diff
@@ -33,7 +33,8 @@ def inner(self, *args, **kwargs):
         node = args[0]
         if self.truncate_entries_flag:
             base_version = self.params.get('scylla_version', default='')
-            system_truncated = True if parse_version(base_version) >= parse_version('3.1') and not is_enterprise(base_version) else False
+            system_truncated = True if parse_version(base_version) >= parse_version(
```

Comment:
convert to simple `if..else` statement

Reply:
R: 36,12: The if statement can be replaced with 'var = bool(test)' (simplifiable-if-statement)